### PR TITLE
Fix Profile - Hobby - Textarea resize issue

### PR DESCRIPTION
**Pull request by gets0ul**
Fix for issue #1197

### CODE REVIEW

File: src/shared/components/Settings/Profile/Hobby/styles.scss [hunk 1]
```diff
@@ -148,6 +148,11 @@
     margin-top: 6px;
   }
 
+  textarea {
+    resize: vertical;
+    max-width: 100%;
```
[the remark this answers is not in the document]
Comment (ThomasKranitsas) [reply src/shared/components/Settings/Profile/Hobby/styles.scss]: @gets0ul ☝️

**Comment by ThomasKranitsas** [comment]
I'm still able to reproduce the issue when focusing on the textarea

File: src/shared/components/Settings/Profile/Hobby/styles.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (gets0ul) [reply src/shared/components/Settings/Profile/Hobby/styles.scss]: Well, actually with resize being vertical only, it does not matter either max-width or min-width.

[the remark this answers is not in the document]
Comment (ThomasKranitsas) [reply src/shared/components/Settings/Profile/Hobby/styles.scss]: Hmm... I think the only way to reproduce the issue is to install grammarly

**Comment by gets0ul** [comment]
Can you share the screencast when focusing the textarea?
Because I'm unable to reproduce it. The textarea size is just fine, it is not shrinking or enlarging when focused.

**Comment by ThomasKranitsas** [comment]
here you go http://take.ms/PpGNs

File: src/shared/components/Settings/Profile/Hobby/styles.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (gets0ul) [reply src/shared/components/Settings/Profile/Hobby/styles.scss]: Ok let me install the grammarly first.

[the remark this answers is not in the document]
Comment (gets0ul) [reply src/shared/components/Settings/Profile/Hobby/styles.scss]: I have it on. And you were right, it's grammarly.
The textarea will be just shrinking when focused.
So what we want to do here? Search a workaround or disable grammarly on this particular textarea field?

[the remark this answers is not in the document]
Comment (ThomasKranitsas) [reply src/shared/components/Settings/Profile/Hobby/styles.scss]: Changing:
```
&.row-2 {
  width: 100%;
}
```
to

```
&.row-2 {
  min-width: 100%;
}
```
solves the issue

[the remark this answers is not in the document]
Comment (gets0ul) [reply src/shared/components/Settings/Profile/Hobby/styles.scss]: There you go...  
So what I suppose to do now with the PR?

[the remark this answers is not in the document]
Comment (ThomasKranitsas) [reply src/shared/components/Settings/Profile/Hobby/styles.scss]: make the changes I suggested above and I'll merge it ;)

**Pull request merged**